docs/7-dashboard-ui/3-Zarf-deployment-ui.md

```diff
@@ -1,16 +1,16 @@
 # Zarf Deployment Web UI
 
-Zarf has a Deployment Web UI built in that supports a number of Zarf features used during the package deployment process. For users who prefer not to use the command line tool, the Web UI creates a simple experience to deploy and manage Zarf clusters and packages. The Web UI can be used to connect to existing clusters (via a Kubeconfig), initialize a cluster, deploy packages into a cluster, update packages in the cluster, and remove packages from the cluster. 
+Zarf has a Deployment Web UI built in that supports a number of Zarf features used during the package deployment process. For users who prefer not to use the command line tool, the Web UI creates a simple experience to deploy and manage Zarf clusters and packages. The Web UI can be used to connect to existing clusters (via a Kubeconfig), initialize a cluster, deploy packages into a cluster, update packages in the cluster, and remove packages from the cluster.
 
-The Zarf Web UI mirrors the functionality of the Zarf CLI commands, but with a more intuitive flow and familiar web application patterns for non-technical users. The web UI does not offer any additional commands or core functionality to Zarf. 
+The Zarf Web UI mirrors the functionality of the Zarf CLI commands, but with a more intuitive flow and familiar web application patterns for non-technical users. The web UI does not offer any additional commands or core functionality to Zarf.
 
 ## Open the Zarf Deployment Web UI
 
-The Zarf Deployment Web UI can easily be spun up with a single command from the CLI. 
+The Zarf Deployment Web UI can easily be spun up with a single command from the CLI.
 
 Follow these steps to get started using the Web UI
 
-1. Step one: [Install the Zarf binary](/docs/3-getting-started.md)
+1. Step one: [Install the Zarf binary](/docs/3-getting-started/index.md)
```

Comment:
```suggestion
1. Step one: [Installing Zarf](../3-getting-started/index.md#installing-zarf)
```

Reply:
Modified in #1663 (did not fully change since this isn't a tutorial and is a little awkward grammar wise to say "Installing")